src/components/FiltersPanelApplied/FiltersPanelApplied.tsx

```diff
@@ -75,29 +75,29 @@ export const FiltersPanelApplied: React.FC<FiltersPanelAppliedProps> = ({
         }, appliedMap[tr('State')] || []);
     }
 
-    if (queryState.issuer.length && issuers?.length) {
+    if (queryState?.issuer.length && issuers?.length) {
         appliedMap[tr('Issuer')] = queryState.issuer.map((u) => issuersMap[u]?.name).filter(Boolean) as string[];
     }
 
-    if (queryState.owner.length && owners?.length) {
+    if (queryState?.owner.length && owners?.length) {
         appliedMap[tr('Owner')] = queryState.owner.map((u) => ownersMap[u]?.name).filter(Boolean) as string[];
     }
 
-    if (queryState.participant.length && participants?.length) {
+    if (queryState?.participant.length && participants?.length) {
         appliedMap[tr('Participant')] = queryState.participant
             .map((u) => participantsMap[u]?.name)
             .filter(Boolean) as string[];
     }
 
-    if (queryState.project.length && projects?.length) {
+    if (queryState?.project.length && projects?.length) {
         appliedMap[tr('Project')] = queryState.project.map((p) => projectsMap[p]?.title).filter(Boolean);
     }
 
-    if (queryState.tag.length && tags?.length) {
+    if (queryState?.tag.length && tags?.length) {
         appliedMap[tr('Tag')] = queryState.tag.map((t) => tagsMap[t]?.title).filter(Boolean);
     }
 
-    if (queryState.estimate.length) {
+    if (queryState?.estimate.length) {
```

Comment:
`const { priority = [], state = [], stateType = '', issuer, owner, participant, project, tag, estimate } = queryState || {};`

wouldn't it be so much nicer to read?